spyder/plugins/editor/widgets/editorstack/helpers.py

```diff
@@ -163,8 +163,10 @@ def filename(self, value):
     def text_changed(self):
         """Editor's text has changed."""
         self.default = False
+        all_cursors = self.editor.all_cursors
+        positions = tuple(cursor.position() for cursor in all_cursors)
         self.text_changed_at.emit(self.filename,
```

Comment:
```suggestion
        self.text_changed_at.emit(self.filename, positions)
```

No need to break this call in two lines.